ddtrace/tracer.py

```diff
@@ -54,6 +54,11 @@ def _parse_dogstatsd_url(url):
 
 
 def _is_agentless_environment():
```

Comment:
It'd probably be a good idea to move this as a static method for the `Tracer` class. Worst case scenario, somebody can more easily override it if necessary — and it makes sense logically though.